spec/menu_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'menu.rb'
+describe Menu do
+  subject(:menu) { described_class.new }
+  context "#initialize" do
```

Comment:
`context` introduces a description of a situation, e.g. "when the user is not logged in". `describe` announces that the next tests are testing a certain method or class.